data-models/pkg/model/health_event_extentions.go

```diff
@@ -38,8 +38,8 @@ const (
 )
 
 type OperationStatus struct {
-	Status  Status `bson:"status"`
-	Message string `bson:"message,omitempty"`
+	Status  Status `bson:"status" json:"status,omitempty"`
+	Message string `bson:"message,omitempty" json:"message,omitempty"`
 }
 
 type HealthEventStatus struct {
```

Comment:
I was thinking we could just stuff this exact object to be the status for example